src/core/appinterface.h

```diff
@@ -208,6 +209,8 @@ class AppInterface : public QObject
      */
     Q_INVOKABLE QObject *positioning() const;
 
+    Q_INVOKABLE QObject *createHttpRequest( bool autoDelete = false ) const;
```

Comment:
Documentation (this one is extra important as we'll want a really exhaustive documentation if this over at https://api.qfield.org/)